### ISSUE TITLE
Multiple multicore executions overlap 

### ISSUE THREAD

**Issue by Nortamo** [openpmix/prrte#1248]
## Background information
Simultaneous executions requesting multiple cores overlap. 
Wondering if this is expected or a bug. 

- prrte version v2.0.2
- pmix version v4.1.2

### Please describe the system on which you are running

* Operating system/version:
RHEL 7.9 
* Computer hardware:
* ( Bull Sequana XH2000)
AMD EPYC 7402 (dual socket) 
* Network type:
InfiniBand HDR200

-----------------------------

## Details of the problem
When launching multiple executions using `prun` the CPU binds overlap. 

```shell
shell$ prun -np 1 --map-by node:pe=4 ./a.out
```
Would be placed on cores [0-3].
If another process is launched with the same settings (while the first one is still running) it will be placed on cores [1-4]. Instead of some non-overlapping range.  Otherwise the affinity mapping works perfectly. 

_Full:_
```shell
prun -np 1 --map-by node:pe=4 --display map-devel ./a.out &
prun -np 1 --map-by node:pe=4 --display map-devel ./a.out &
```
->
```
process on node1 with cores Cpus_allowed_list: 0-3,128-131
process on node1 with cores Cpus_allowed_list: 1-4,129-132
```

Is this expected behavior or am I perhaps missing some option or configuration to make the placement non-overlapping ? Based on the `--display map-devel` output multiple cpus/proc does not seem to affect the number of slots used. 

Same behavior is observed when running locally and using slurm (21.08) integration (prrte started within a job allocation and srun starts prte on each node).

**Comment by rhc54** [comment]
Hmmm...that is clearly a bug. Will investigate.

**Comment by rhc54** [comment]
Found the bug - still working on it.

**Comment by rhc54** [comment]
Sorry for the lengthy delay - retirement has its privileges :-)

This is fixed in #1383
